.github/workflows/deploy.yml

```diff
@@ -0,0 +1,25 @@
+name: Deploy to Droplet
+'on':
+  push:
+    branches:
+      - main
```

Comment:
This gotta be `master` branch

Reply:
That's a touchy subject but I fixed it lol